pages/linux/wtype.md

```diff
@@ -0,0 +1,28 @@
+# wtype
+
+> Simulate keyboard input on Wayland, similar to `xdotool type` for X11.
+> More information: <https://github.com/atx/wtype>.
+
+- Type text:
+
+`wtype "{{Hello World}}"`
+
+- Type a specific key:
+
+`wtype -k {{Left}}`
+
+- Press a modifier (e.g., "shift", "ctrl"):
+
+`wtype -M {{ctrl}}`
+
+- Release a modifier:
+
+`wtype -m {{ctrl}}`
+
+- Delay between keystrokes (in milliseconds):
```

Comment:
```suggestion
- Wait between keystrokes (in milliseconds):
```
It needs to be in imperative mood(style guide). Let me know if I am wrong and it is in imperative mood already, I am not native english speaker.